### PR TITLE
Update HTTP gateway docs to add GCP support

### PR DESCRIPTION
**Pull request by amarthadan**
The v0.5 docs are just a copy of v0.4 so I guess there are a lot of things to be fixed, I just needed a place to document 0.5 changes.

### CODE REVIEW

**Comment by wkande** [comment]
@amarthadan I am not sure what this is. I did not plan on adding the `v.0.5` docs directory until the release of `v0.4` in the next few day. I usually make the next release (v0.5) after final cleanup of links (and several other modes) in the newly released
version (v0.4). I fear we might be on a major collision course.

It appears you made a base for v0.5 so that you could the change a few things for GCP Gateway. 

v0.5 documentation tasks should be held in the Documents column until I get v0.4 deployed. I do not know when Burak will release but I was planning on him saying this week and as such I would release v0.4 of the dos this week.

If you can hold off doing any more v0.5 doc work we can move the changes of the few files you have made [here](https://github.com/api3dao/api3-docs/pull/561/commits/a19013be01f8b90af9d9f728154e9252cc21d101) over (manually) to my base copy for v0.5 which will be current with v0.4.

**Comment by amarthadan** [comment]
@wkande Sorry, I didn't want to mess up your flow and plans. Feel free to hold off with this as long as you need, I just needed a place where I can introduce the 0.5 changes. You can throw out the first commit (introducing the 0.5 directories) and apply only the second one (HTTP gateway changes) once that directory is properly set up.

**Comment by wkande** [comment]
No problem. I think we are deploying `v0.4` soon. Burak told me a few days ago this might be the week. I will merge your changes to my `v0.5` base. Right now Martin and I are doing content reviews before doing so.

File: docs/airnode/v0.5/grp-providers/tutorial/quick-deploy-gcp/README.md [hunk 1]
```diff
@@ -171,23 +171,106 @@ docker run -it --rm ^
...
+## Test the Airnode
+
+After a successful deployment the Airnode can be tested directly using the
+[HTTP Gateway](../../guides/build-an-airnode/http-gateway.md) without accessing
+the blockchain. You provide endpoint parameters to get a response from an
+integrated API.
+
+### HTTP Gateway
+
+Looking at the config.json shows that the HTTP Gateway was activated for our
```
Comment (Siegrift) [inline docs/airnode/v0.5/grp-providers/tutorial/quick-deploy-gcp/README.md:183]: this is strange wording

Comment (wkande) [reply docs/airnode/v0.5/grp-providers/tutorial/quick-deploy-gcp/README.md:183]: I will make note of this and update if needed when I do the merge.

Comment (wkande) [reply docs/airnode/v0.5/grp-providers/tutorial/quick-deploy-gcp/README.md:183]: Looking at the config.json code snippet below shows the HTTP Gateway was
activated for our Airnode. Furthermore the endpoint for `/coins/{id}` with an
`endpointId` of `0xf...53c` is set to be `testable:true`. Each individual
`endpointId` in `triggers.rrp[n]` must be marked as `testable: true || false` to
allow for the desired access.

**Comment by wkande** [comment]
This PR is replaced with PR: 568.

This PR will not be merged to the main branch.

**Pull request closed without merging**